liquidity/liquidity.go

```diff
@@ -0,0 +1,125 @@
+// Package liquidity is responsible for monitoring our node's liquidity. It
+// allows setting of a liquidity rule which describes the desired liquidity
+// balance on a per-channel basis.
+package liquidity
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"strings"
+	"sync"
+
+	"github.com/lightningnetwork/lnd/lnwire"
+)
+
+var (
+	// ErrNoParameters is returned when a request is made to lookup manager
+	// parameters, but none are set.
+	ErrNoParameters = errors.New("no parameters set for manager")
```

Comment:
Maybe we can just start with an empty list of rules and avoid this special case?